cmd/config/cmd.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/cmd/config/get"
+	"github.com/openshift/rosa/cmd/config/set"
+	"github.com/openshift/rosa/pkg/config"
+)
+
+func configVarDocs() (ret string) {
+	configType := reflect.ValueOf(config.Config{}).Type()
+	fieldHelps := make([]string, configType.NumField())
+	for i := 0; i < len(fieldHelps); i++ {
+		tag := configType.Field(i).Tag
+		name := strings.Split(tag.Get("json"), ",")[0]
+		doc := tag.Get("doc")
+		fieldHelps[i] = fmt.Sprintf("\t%-15s%s", name, doc)
+	}
+	ret = strings.Join(fieldHelps, "\n")
+	return
```

Comment:
Having named return values is not something we encourage in the CLI. 

I dont think it adds any benefit to this func in particular, when we can just `return strings.Join(fieldHelps, "\n")

Reply:
Returned as suggested